vehicles.go

```diff
@@ -91,3 +92,14 @@ func (c *Client) Vehicle(vehicleID int64) (*Vehicle, error) {
 	resp.Response.c = c
 	return resp.Response, nil
 }
+
+// CommandPath returns the URL for a provided command.
+func (v *Vehicle) CommandPath(command string) string {
+	// The wake command is not rooted at .../command.
+	parts := []string{v.c.baseURL, "vehicles", strconv.FormatInt(v.ID, 10)}
+	if command != "wake_up" {
```

Comment:
This seems a bit unexpected here.  Perhaps there should be a couple different functions

1. Something that provides the base path (for use by the next 2 and also wake_up?)
2. Something that works for commands
3. Something that works for fetching state (like already exists)

Reply:
I‘d probably just not use commandPath for wakeup then and add a comment to that effect